pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -230,6 +231,8 @@ func manageClientCABundle(lister corev1listers.ConfigMapLister, client coreclien
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace, Name: "csr-controller-ca"},
 		// this bundle is what this operator uses to mint new client certs it directly manages
 		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.OperatorNamespace, Name: "managed-kube-apiserver-client-ca-bundle"},
+		// this bundle is what cluster-ingress-operator uses to mint default certificates for routers
+		resourcesynccontroller.ResourceLocation{Namespace: operatorclient.IngressOperatorNamespace, Name: "router-ca"},
```

Comment:
this means that we trust any client cert signed by the ingress CA (and groups and users claimed inside it). Is this really what we want?